@FISHDATA/plotDist2centerline.m

```diff
@@ -1,7 +1,25 @@
-function plotDist2centerline(obj)
+%  Filename: plotDist2centerline.m (method of FISHDATA class)
+%  Abstract:
+%        Plot distance to centerline of the arena for a FISHDATA.
+%
+%   SYNTAX:
+%       1. obj.plotDist2centerline(phase,mark)
+%       2. plotDist2centerline(obj,phase,mark)
+%
+% - plotDistance2centerline:
+%      - FISHDATA method
+%      - arg1: the entire process / test-phase only / baseline-phase / training-phase
+%      - arg2: pixelSize
+%      - arg3: w/o extinction point (blue triangle)
+%      - arg4: w/o shocking events (red filled-circles)
+%      - arg5: w/o shadows to demarcate consecutive phases
+% TODO: Pixels converted to mm may still have problems.
+%       When use new data, Change 29399 to 29400(the last frame num)
+function plotDist2centerline(obj,...FISHDATA object
+    phase,...  % 0: Entire experimental phases;  1:Baseline; 2:Training 4:Test
+    mark)  % mark is a matrix of any combination of "shadows","shock" and "extinction point" 
```

Comment:
It would be better to use **varagin** to replace this mark matrix. Please the ABLITZER method findFish and the MATLAB documentation to learn its usage.
@danyang123